framework/auth/__init__.py

```diff
@@ -60,6 +63,13 @@ def register_unconfirmed(username, password, fullname):
             password=password,
             fullname=fullname)
         user.save()
+        mails.queue_mail(
+            to_addr=user.username,
+            mail=mails.NO_ADDON,
+            send_at=datetime.utcnow() + settings.NO_ADDON_WAIT_TIME,
+            user=user,
+            fullname=user.fullname
```

Comment:
```user``` is not available in the mako context. It could be added, but ```fullname``` is the only attribute used.